languages/go/exercises/concept/conditionals/.docs/hints.md

```diff
@@ -6,48 +6,62 @@ Conditionals are used to check for certain conditions and/or criteria. The most
 
 ### 1. Multiple cases (else if)
```

Comment:
The names of the task-based hint headers in this file don't match up with the names used in the `instructions.md` file.

E.g. this heading should be:

```suggestion
### 1. Calculate the score of any given card.
```